### ISSUE TITLE
Go version of dashboard uses a lot of memory

### ISSUE THREAD

**Issue by chrisvdg** [maxux/rtinfo-dashboard#7]
Noticed that the rtinfo dashboard is taking ~16Gig of my system's memory.

![image](https://user-images.githubusercontent.com/23582741/52053036-829a2280-2558-11e9-9b04-f3c8a36853f7.png)

Branch used `iops`

Container was running for 20 days with ~180 nodes attached (TLRE)

Docker image `gigtech/rtinfo-dashboard:0.0.1`

Dockerfile used to create the image
```dockerfile
FROM golang as builder

WORKDIR /go/src/github.com/maxux/

# Build webserver
RUN git clone https://github.com/maxux/rtinfo-dashboard.git && go get -u github.com/jteeuwen/go-bindata/...
RUN cd rtinfo-dashboard/wserver-go && git checkout iops && go get -d -u ./... && go generate && CGO_ENABLED=0 GOOS=linux go build -tags netgo -a -o $GOPATH/bin/dashboard

# Build final image
FROM alpine

COPY --from=builder /go/bin/dashboard /bin

EXPOSE 8091

CMD ["dashboard", "--addr", ":8091", "--endpoint", "http://127.0.0.1:8089/json"]
```

**Comment by maxux** [comment]
@zaibon memory leak ?

**Comment by zaibon** [comment]
Maybe, I'll enable memory profiling on it and see what comes out of it

**Comment by zaibon** [comment]
Seems I had this leak fixes for long time on my branch, but the fix didn't get pulled here

**Comment by zaibon** [comment]
fixed by #8
